lib/caesar.rb

```diff
@@ -1,6 +1,107 @@
+require 'pry'
 class Caesar
-  attr_reader :key
+  attr_reader :cipher
 
   def initialize
+    @cipher = []
```

Comment:
Since you don't use @cipher, you can just get rid of line 6, leaving your initialize empty, so you could just get rid of the whole method